mgmtsystem_action/security/ir.model.access.csv

```diff
@@ -1,5 +1,6 @@
 "id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
 "access_mgmtsystem_action","mgmtsystem.action.user","model_mgmtsystem_action","mgmtsystem.group_mgmtsystem_user",1,1,1,0
+"access_mgmtsystem_claim","mgmtsystem.action.user","crm_claim.model_crm_claim_stage","mgmtsystem.group_mgmtsystem_user",1,0,0,0

```

Comment:
> Are you sure this is needed? AFAIK Actions don't rely on crm_claim.

Tell me if i'm wrong! I did this
* Installed the mgmtsystem_action module
* Created a new user
* Gave it User permissions
![captura de pantalla de 2015-07-28 16-38-09](https://cloud.githubusercontent.com/assets/7594716/8945728/4d81701c-3547-11e5-8998-cfe653dc6d30.png)
* Logged in with the user
* Went to Management System > Management System > Actions > Create
* Error popped up
![captura de pantalla de 2015-07-28 16-43-20](https://cloud.githubusercontent.com/assets/7594716/8945795/e67a21c4-3547-11e5-92f8-c84979986563.png)